plugin-flex-ts-template-v2/src/utils/serverless/TaskRouter/TaskRouterService.ts

```diff
@@ -44,11 +46,88 @@ interface UpdateWorkerChannelResponse {
   workerChannelCapacity: WorkerChannelCapacityResponse;
 }
 
+const STORAGE_KEY = 'pending_task_updates';
```

Comment:
can we make the storage key unique to the flex session - otherwise weird behavior will appear when people have multiple sessions open

Reply:
```javascript
private flex_service_instance_sid = Manager.getInstance().serviceConfiguration.flex_service_instance_sid;
const STORAGE_KEY = `peending_task_updates_${this.flex_service_instance_sid}`;
```